### PR TITLE
Phomiuna Acqueducts - Fixed Oil Lamps and Doors

### PR DESCRIPTION
**Pull request by Solarsurge**
*Entire east side's doors weren't coded
*Updated descriptions to better identify
*Fixed two broken oil lamp animations
*East door now properly opens

**_I affirm:_**
- [X] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rm.lua [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
--- Area: Phomiuna_Aqueducts
---  NPC: _0rm (Oil lamp)
--- Notes: Opens South door at J-7 from inside.
--- !pos -63.703 -26.227 83.000 27
------------------------------------
-require("scripts/globals/missions");
+--    Area: Phomiuna_Aqueducts
+--  NPC: Oil Lamp - North (West)
+-- Opens Door at F-7 from inside.
+--  ID: 16888052 !pos -63 -26 83
```
Comment (ibm2431) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rm.lua]: NPC IDs can/do shift, so these comments might fall out of date and confuse people later.

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua [hunk 2]
```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+--    Area: Phomiuna_Aqueducts
+--  NPC: Oil Lamp - Water (East)
+-- ID: 16888080  !pos 104 -26 77
+-----------------------------------
+require("scripts/globals/missions");
```
Comment (ibm2431) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua]: These mission requires aren't being used for these lamps...!

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    player:messageSpecial(ID.text.LAMP_OFFSET+2); -- water lamp
+    npc:openDoor(7); -- lamp animation
+
+    local element = VanadielDayElement();
+    -- printf("element: %u",element);
+
+    if (element == 2) then -- waterday
+        if (GetNPCByID(DoorOffset+7):getAnimation() == 8) then -- lamp fire open?
+            GetNPCByID(DoorOffset-2):openDoor(15); -- Open Door _0rk
```
Comment (ibm2431) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua]: Comment says these eastern doors open up _0rk, but I think they _might_ open a different door, and the erroneous comment was a result of a copy-paste :wink:

**Review by ibm2431 — CHANGES_REQUESTED** [review]
Logic looks okay as far as I can tell, assuming the lamp IDs are in the same order as the other set.

Requested changes are just comment-related and removing unused requires (admittedly, you were sabotaged by the existing lamps).

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rm.lua (hunk 1 above)
Comment (Solarsurge) [reply scripts/zones/Phomiuna_Aqueducts/npcs/_0rm.lua]: IDs deleted.

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua (hunk 2 above)
Comment (Solarsurge) [reply scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua]: Mission requires deleted. They were copied/pasted from west side lamps.

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua (hunk 3 above)
Comment (Solarsurge) [reply scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua]: Corrected. It's door _0rl

**Review by ibm2431 — APPROVED** [review]
Thanks, Solarsurge!

**Pull request merged**